Modules/Loadable/Sequences/Widgets/qMRMLSequenceBrowserPlayWidget.h

```diff
@@ -58,13 +62,17 @@ class Q_SLICER_MODULE_SEQUENCES_WIDGETS_EXPORT qMRMLSequenceBrowserPlayWidget
   /// Add a keyboard shortcut for next frame button
   void setNextFrameShortcut(QString keySequence);
 
+  /// Get recordingControlsVisible value
```

Comment:
```suggestion
  /// Returns true if recording controls (record and snapshot buttons) are allowed to be shown.
  /// Note that regardless of this flag, recording controls are not shown if recording is not enabled
  /// for any of the browsed sequences.
```

Reply:
Corresponding change is proposed in https://github.com/Slicer/Slicer/pull/5897